app/src/main/res/values/strings.xml

```diff
@@ -11,6 +11,7 @@
     <string name="dialog_delete">Delete</string>
     <string name="dialog_remove">Remove</string>
     <string name="dialog_deny">Cancel</string>
+    <string name="dialog_confirm">Confirm</string>
```

Comment:
I would use "Enable" here, to make clear that's what is going to happen when pressed.